src/sentry/eventstore/snuba/backend.py

```diff
@@ -34,14 +31,24 @@ def get_events(
         """
         Get events from Snuba.
         """
+        assert filter
+
+        filter_keys = {}
+        if filter.project_id:
+            filter_keys["project_id"] = filter.project_id
+        if filter.group_id:
+            filter_keys["group_id"] = filter.group_id
+        if filter.event_id:
+            filter_keys["event_id"] = filter.event_id
+
         cols = self.__get_columns(additional_columns)
 
         result = snuba.raw_query(
-            start=start,
-            end=end,
-            selected_columns=cols,
-            conditions=conditions,
+            start=filter.start,
+            end=filter.end,
+            conditions=filter.conditions,
             filter_keys=filter_keys,
+            selected_columns=cols,
```

Comment:
nit: let's move selected cols before filter keys so that it follows the order of the clauses in an SQL expresison ?

Reply:
Done